### ISSUE TITLE
How do I turn off the auto updater?

### ISSUE THREAD

**Issue by ASmith-** [nextcloud-snap/nextcloud-snap#1871]
My running instance has suddenly returned to the horrific bug in the sql database that fills up the netcloud snap log with unable to show images errors suddenly popping up. 19 days ago I encountered this in nextcloud 22.1.1snap1 Rev 28503 latest stable nextcloud and had to issue the command to revert back to 21.0.4 snap2. Now, I've had to do it again. Does the auto-updater have a disable setting a user can reach inside the snap installed nextcloud or is this buried in the auto-start widgets in the file system?

**Comment by pachulo** [comment]
No, you cannot stop it from updating indefinitely. What you can do is use one of the channels, like `21/stable`, so you don't get major updates till you move to the next one.

By the way, is your issue like this one? https://github.com/nextcloud-snap/nextcloud-snap/issues/1873
